packages/ts/react-crud/src/autoform.tsx

```diff
@@ -86,7 +88,7 @@ export function ExperimentalAutoForm<TItem>({
         ) : null}
         <Button
           theme="primary"
-          disabled={disabled}
+          disabled={disabled || (isEditing && !form.dirty)}
```

Comment:
Well, TBH, first I implemented it exactly as suggested here (without a state), but when reading the code `item !== emptyItem` doesn't announce what is the logic. Then after introducing a local variable, I recalled that while working with component like `<AutoForm />` it would be very helpful to also have some state such as `isNew` or `isEdit` which we can  expose later. The logic that determines the value for `isEditing` state is something we can discuss further to cover more cases I haven't think of.

Reply:
Sure, extracting something with a name for readability makes sense. I see that you are now using a function. You can change that to just be a local variable, the function is just overhead.

The name also threw me off a bit yesterday, I didn't associate that it means that it's currently editing an existing item.

Reply:
Done.